online-shop/src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './App.css';
+
+import ListProducts from "./components/listProducts/listProducts.component";
+
+
+function App() {
+  return (
+  
+
+    <div className="App">
+      
```

Comment:
Here you will have to wrap it with the `Theme`, the `CssBaseline` reseter and the actual router with the `Switch` logic for the routes